tests/components/smhi/test_config_flow.py

```diff
@@ -16,6 +17,8 @@
 
 from tests.common import MockConfigEntry
 
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Comment:
we still have some inline patches for the async setup entry laying around

Reply:
Only one but it's left there to test it's called as it's supposed to.